src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -665,35 +665,31 @@ void PeakDetectorCLI::saveCSV(string setName) {
     #endif
 
 	string fileName = mavenParameters->outputdir + setName + ".csv";
-
-    CSVReports* csvreports = new CSVReports(mavenParameters->samples);
-    csvreports->setMavenParameters(mavenParameters);
-
+	if(fileName.empty()) return;
+	
     if (mavenParameters->allgroups.size() == 0 ) {
 		cout << "Writing to CSV Failed: No Groups found" << endl;
         return;
     }
 
-    if(fileName.empty()) return;
-    
-    if (mavenParameters->samples.size() == 0) return;
+    vector<mzSample*> samples = mavenParameters->samples;
+    if ( samples.size() == 0) return;
 
-    csvreports->setUserQuantType(quantitationType);
+    MavenParameters* mp = mavenParameters;
```

Comment:
@IpankajI Why are we making a new variable mp here?

Reply:
just for consistency and better readability of code, anyway it's a local pointer variable so there won't be any memory hit.

Reply:
@IpankajI Now there are two variables that can be used for the same thing. mp and mavenParameters. Future contributors could use either of them. It will be more consistent and readable if we use mavenParameters.